Implement change password for refugee and fix it for host and admin as well

mihaipopescu opened the pull request:
### What does it fix?

Addresses #66 to enable users to change their passwords but since it's reusing existing implementation found for the host/admin, it differs from the mockup described in #66.

Later I discovered that the existing implementation was broken so the PR addresses and fixes the password reset for host and admin as well.

Also, I've only added Romanian translation. I could have used Google Translate for the other languages but I think it's better the translators add a proper translation.

### How has it been tested?

Locally.

mihaipopescu committed:
implement reset-password for refugee

The implementation is an adaptation from existing code found for host.

mihaipopescu committed:
revert mistake

mihai-unity commented:
![Screen Shot 2022-03-07 at 21 49 05](https://user-images.githubusercontent.com/57262907/157115612-5b0eda30-8730-4e8b-bff0-c6294d1613c6.png)

@anastk-code4 Would the current implementation be acceptable? Would you like me to change the translation to match the mockup?

mihaipopescu commented:
![Screen Shot 2022-03-07 at 21 49 05](https://user-images.githubusercontent.com/272470/157116021-99793b23-6570-4686-a8ff-e7920bad0ade.png)

@anastk-code4 Would the current implementation be acceptable? Would you like me to change the translation to match the mockup?

mihaipopescu committed:
add romanian translation lost after merge

mihaipopescu committed:
fix route after merge

mihaipopescu commented:
something's still off. I can confirm the password is changed but to some wrong value since I can't login after logging out, neither with the new password nor with the old one. 🤔

mihaipopescu committed:
fix for password reset

newPassword was an empty value and that made me wonder whether the code
was using the wrong key and since it was simpler renaming that instead
of changing everything else to use newPassword I just did that.

mihaipopescu committed:
ignore artifacts after make install

mihaipopescu commented:
ok, I found some time to debug the previous issue and figured it out.

mihaipopescu commented:
@anastk-code4 can you verify if this implementation works for you?
